src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource.cs

```diff
@@ -89,16 +89,6 @@ private sealed partial class TagSource : ForegroundThreadAffinitizedObject
             /// </summary>
             private readonly CancellationTokenSource _initialComputationCancellationTokenSource = new CancellationTokenSource();
 
-            /// <summary>
-            /// Whether or not we've gotten any change notifications from our <see cref="ITaggerEventSource"/>.
-            /// The first time we hear about changes, we fast track getting tags and reporting 
-            /// them to the UI.
-            /// 
-            /// We use an int so we can use <see cref="Interlocked.CompareExchange(ref int, int, int)"/> 
-            /// to read/set this.
-            /// </summary>
-            private int _seenEventSourceChanged;
```

Comment:
this value (and how it was used) was completely pointless.  We tried to fast-track the first event we heard.  However, that's completely unnecessary as the very last thing the TagSource constructor does (see line https://github.com/dotnet/roslyn/pull/46955/files#diff-e80af3f0cd74d09b03f70e25b96d971fL136) is to kick off a preemptive tagging of the document to ensure results show up fast.  this just added complexity and more work happening (and contending) when a file loaded.